tethys_gizmos/gizmo_options/map_view.py

```diff
@@ -293,11 +302,13 @@ class MVLayer(SecondaryGizmoOptions):
         options (dict, required): A dictionary representation of the OpenLayers options object for ol.source.
         legend_title (str, required): The human readable name of the layer that will be displayed in the legend.
         layer_options (dict): A dictionary representation of the OpenLayers options object for ol.layer.
+        editable (bool): If true the layer will be editable with the tethys_map_view drawing/editing tools.
         feature_selection (bool): Set to True to enable feature selection on this layer. Defaults to False.
         geometry_attribute (str): The name of the attribute in the shapefile that describes the geometry
         legend_classes (list): A list of MVLegendClass objects.
         legend_extent (list): A list of four ordinates representing the extent that will be used on "zoom to layer": [minx, miny, maxx, maxy].
         legend_extent_projection (str): The EPSG projection of the extent coordinates. Defaults to "EPSG:4326".
+        tethys_toc (bool): If true the layer will be included in the Table of Contents gizmo
```

Comment:
I don't like the idea of the MVLayer object knowing anything about the Table of Contents Gizmo. Why not just pass the MVLayer objects that you want shown on the table of contents to the table of contents when you create it?